ckanext/portalopendatadk/dcat_profile.py

```
@@ -884,17 +950,36 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):
+
+                    import re
+
+                    if (
+                        isinstance(val, str)
+                        and re.fullmatch(r"[0-9A-Fa-f]+", val)
+                        and len(val) % 2 == 0
+                    ):
+                        literal = Literal(val, datatype=XSD.hexBinary)
+                    else:
+                        log.warning(
+                            f"Invalid checksumValue '{val}' – storing as plain string"
```

Ruff (0.12.2)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)